markdown/events/2023/hackathon-march-2023/spain-crg.md

```diff
@@ -18,6 +18,68 @@ Primary contacts: [<i class="fab fa-slack"></i> Björn Langer](https://nfcore.sl
 
 Please use the [main registration form](https://nf-co.re/events/2023/hackathon-march-2023) to sign up, but feel free to contact Björn, Luca or Toni if you would like any more information.
 
+## Local Schedule
+
+<div class="table-responsive">
+    <table class="table table-hover table-sm table-bordered">
+        <thead>
+            <tr>
+                <th>Time</th>
+                <th>Mon. 27 Mar., 2023</th>
+                <th>Tue. 28 Mar., 2023</th>
+                <th>Wed. 29 Mar., 2023</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td data-timestamp="1679902200" data-timeformat="HH:mm z">09:30</td>
+                <td background-color:navy; rowspan="2">Local Welcome and Setup</td>
```

Comment:
```suggestion
                <td style="color: pearl; background-color:navy;" rowspan="2">Local Welcome and Setup</td>
```
Only this way this syle is applied (need to set teh color to white to have the text also readable in the light theme).